deployment/dev/devenv.sh

```diff
@@ -3,8 +3,15 @@
 cd $(dirname $0)
 cd compose
 
-# Build local images first
-bazel run //:build-images
+if [ -z "$BUILD" ]; then
+  echo "Using existing images. To rebuild, run:"
+  echo ""
```

Comment:
why the `echo ""` ?

Reply:
Visually more distinguishable in the logs